incubator/elasticsearch/README.md

```diff
@@ -31,66 +31,122 @@ The command deploys Elasticsearch on the Kubernetes cluster in the default confi
-| `metrics.resources`             | Metrics exporter resource requests/limit              | `requests: { cpu: "25m" }`                                          |
-
-The above parameters map to the env variables defined in [bitnami/elasticsearch](http://github.com/bitnami/bitnami-docker-elasticsearch). For more information please refer to the [bitnami/elasticsearch](http://github.com/bitnami/bitnami-docker-elasticsearch) image documentation.
+|            Parameter                              |                      Description                                                                                        |                               Default                               |
+|---------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------|
+| `image.registry`                                  | Elasticsearch image repository                                                                                            | `docker.io`                                                         |
+| `image.repository`                                | Elasticsearch image name                                                                                                  | `bitnami/elasticsearch`                                             |
+| `image.tag`                                       | Elasticsearch image tag                                                                                                   | `{VERSION}`                                                         |
+| `image.pullPolicy`                                | Image pull policy                                                                                                         | `Always`                                                            |
+| `image.pullSecrets`                               | Specify image pull secrets                                                                                                | `nil`                                                               |
+| `name`                                            | Elasticsearch cluster name                                                                                                | `elastic`                                                           |
+| `serviceAccountName`                              | Service account for Elasticsearch release                                                                                 | `elastic`                                                           |
+| `plugins`                                         | Elasticsearch node plugins                                                                                                | `io.fabric8:elasticsearch-cloud-kubernetes:6.2.3` (required plugin) |
```

Comment:
The default value as per the `values.yaml` in this PR is `http://github.com/fabric8io/elasticsearch-cloud-kubernetes/archive/elasticsearch-cloud-kubernetes-6.2.3.zip`. you need to update it here

Reply:
I'd wait for the plugin to be released.